assets/amp/amp-player-main.js

```diff
@@ -22,6 +22,8 @@ let maxScroll;
 let scaleVal = 1;
 let scalingDown = false;
 let deltaY = 0;
+let previousStories = [];
+let doesHaveMorePages = false;
```

Comment:
Instead of these many variables can we use a single object?

Reply:
Resolved in latest commits